style.css

```diff
@@ -1,464 +1,1083 @@
+  font-size: 20px;
+  line-height: 24px;
+}
+
+.skill li {
+  float: left;
+  padding: 4px 12px;
+  margin: 4px;
+  background: #ebebff;
+  border-radius: 8px;
+}
+
+.project-button {
+  /* padding-left: 5%; */
+  padding-bottom: 12px;
+}
+
+.project-button button {
+  width: 126px;
+  height: 48px;
+  background-color: transparent;
+  border-color: #6070ff;
+  color: #396df2;
+  border-radius: 8px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+}
+
+.skill li button {
+  color: #6070ff;
+  background-color: #ebebff;
+  border-color: transparent;
+  font-style: normal;
+}
+
+.project-button button:hover {
+  background-color: lightblue;
+  cursor: pointer;
+}
+
+.about {
+  display: flex;
+  flex-direction: column;
+  margin-top: 50px;
+  background-color: white;
+  border-radius: 0 50px 0 0;
+  gap: 12px;
+}
+
+.sct-1 {
+  display: flex;
+  flex-direction: column;
+}
+
+.sct-1 h2 {
+  margin-bottom: 12px;
+  margin-top: 40px;
+  margin-left: 25px;
+  font-style: normal;
+  font-size: 40px;
+  color: #172b4d;
+}
+
+.sct-1 > p {
+  margin: 0 24px 12px 4.5%;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.sct-1 > a {
+  margin: 5px 0 26px 4.5%;
+  color: #7f8cff;
+}
+
+.sct-2 {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  margin-top: 3.5%;
+
+  /* margin-left: 7% */
+}
+
+.abt-social {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  list-style: none;
+  padding-bottom: 10px;
+}
+
+.abt-social ul {
+  margin-top: 26px;
+  display: flex;
+}
+
+.abt-social li {
+  display: inline-block;
+  margin-left: 4.5%;
+}
+
+.sct-1 button:hover {
+  background-color: lightblue;
+  cursor: pointer;
+}
+
+#line-1 {
+  border-left: 10px solid #dfe1e6;
+  background-color: #dfe1e6;
+  width: 94%;
+}
+
+.arrow {
+  margin-left: 7%;
+  display: flex;
+  width: 80%;
+  justify-content: space-between;
+  align-items: center;
+  padding: 1rem 0;
+}
+
+.tech {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  margin: 15px;
+  color: #253858;
+}
+
+.languages li {
+  margin-left: 7%;
+  background: #f7f7f9;
+  width: 110%;
+}
+
+.form-container {
+  position: relative;
+  background-color: #6070ff;
+  margin-top: 177px;
+  border-top-left-radius: 80px;
+  display: flex;
+  flex-direction: column;
+}
+
+.back-spot {
+  position: absolute;
+  z-index: 1;
+  top: 21px;
+  right: 0;
+}
+
+.form-bottom {
+  z-index: 99;
+}
+
+.form-bottom h2 {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  margin-top: 103px;
+  color: white;
+}
+
+.form-bottom p {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 28px;
+  margin-top: 12px;
+  margin-left: 24px;
+  margin-right: 24px;
+  color: white;
+  text-align: center;
+}
+
+.form-bottom form {
+  display: flex;
+  flex-direction: column;
+}

```

Comment:
- [ ] [OPTIONAL] The previous reviewer forgot to mention this but the form elements aren't aligned as they're shown in the Figma design so please make sure you fix that:
    - Your design:
      ![image](https://user-images.githubusercontent.com/104968236/230734018-95dabcc3-460c-447b-8ef8-b79b98709f6a.png)
    - Figma design:
      ![image](https://user-images.githubusercontent.com/104968236/230734058-dcbb7a86-2c7d-47a9-ade1-0c1819f8b637.png)